### PR TITLE
Update GRBAC for VS Code

### PR DESCRIPTION
**Pull request by tsmithv11**

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc [hunk 1]
```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
 
 .. For *GRBAC*, provide the following permissions.
 +
-* *Policies*: 'View' permissions
-* *Projects*: 'View' permissions
-* *Repositories*: ('View' and 'Create') OR ('View' and 'Update') permissions
+* *Policies* -> *Policies*: 'View' permissions
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:43]: Why duplicate  "Policies"? 
State the entity requiring the permissions, the permissions required and then the path.

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc [hunk 2]
```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
...
+* *Application Security* -> *Projects*: 'View' permissions
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:44]: What about all other features besides Application Security? Should we have a note?

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc [hunk 3]
```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
...
+* *Settings* -> *Providers*: ('View' and 'Create') OR ('View' and 'Update') permissions
```
Comment (JBakstPaloAlto) [inline docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:45]: What are "Providers" in this context? Code providers? If so, state the entity requiring the permissions, the permissions required and then the path.

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc (hunk 1 above)
Comment (tsmithv11) [reply docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:43]: One is the header, one is the line
Screenshot 2024-10-27 at 10 43 41 PM

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc (hunk 2 above)
Comment (tsmithv11) [reply docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:44]: The first is the header
Screenshot 2024-10-27 at 10 44 12 PM

File: docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc (hunk 3 above)
Comment (tsmithv11) [reply docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:45]: Screenshot 2024-10-27 at 10 45 48 PM

Comment (JBakstPaloAlto) [reply docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc:45]: I would have the name of the entity requiring the permission, the permission, and finally the path. For example: Providers: CRUD. To configure these permissions, navigate to Settings > Providers, and select the required permissions.

**Comment by JBakstPaloAlto** [comment]
Merge as per @tsmithv11's request of 4 Nov 2024.

**Pull request merged**